AUTHORING_GUIDE.md

```diff
@@ -923,7 +923,7 @@ Add the new environment variables to the `envs` dictionary.
 ```py
 TEST_CONFIG_OVERRIDE = {
     # You can opt out from the test for specific Python versions.
-    "ignored_versions": ["2.7", "3.7", "3.9", "3.10", "3.11"],
+    "ignored_versions": ["2.7", "3.8", "3.10", "3.11", "3.12"],
```

Comment:
did this need to match 3.13 from above?

Reply:
Don't believe so, we want samples to support all those versions but we purely test lowest and highest supported version by default. Does that make sense?